webview/src/experiments/components/table/styles.module.scss

```diff
@@ -637,17 +637,28 @@ $badge-size: 0.85rem;
   font-size: 0.65rem;
 }
 
-.plotBox {
+%iconBox {
   background-color: $checkbox-background;
   border-radius: $checkbox-border-radius;
   border: $checkbox-border-width solid $checkbox-border;
   cursor: pointer;
   display: flex;
   height: $checkbox-size;
   width: $checkbox-size;
+}
+
+.plotBox {
+  @extend %iconBox;
```

Comment:
[Q] Should I be using `cx` instead of extending a base class?

Reply:
Extending a base class sounds like a better option to me since this class isn't conditionally added. 

Though  `%iconBox` should be moved to the "Extendable Silent Rules" section (starts on line 15).